dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -0,0 +1,202 @@
+import { ArticleDesign } from '@guardian/libs';
+
+import { Card } from './Card/Card';
+import { UL } from './Card/components/UL';
+import { LI } from './Card/components/LI';
+
+type Props = {
+	trails: TrailType[];
+};
+
+export const DynamicFast = ({ trails }: Props) => {
+	const primary = trails[0];
+	const secondary = trails[1];
+	const third = trails[2];
+	const groupOne = trails.slice(3, 6);
+	const groupTwo = trails.slice(7, 12);
```

Comment:
Is it worth adding some comments here about what assumptions we make in the current incarnation of this component? That way it's quicker to understand what changes we think need made for the future!

Reply:
Done 👍